### ISSUE TITLE
Progress? Documentation?

### ISSUE THREAD

**Issue by robertjuh** [EddyVerbruggen/nativescript-ar#4]
Hey Eddy, i appreciate your work,

Currently working on an AR project for Android where i have to integrate another THREE.js project into a multiplatform app that preferbly can run as a web app (for my bachelor thesis). Fully web based is technically impossible as far as i know. 

Are you planning on integrating the second preview of ARCore? Any progress on this current project? 
Is it possible to develop on Android? And if so, which platform do you suggest for developing? In your documentation it only says xCode for apple.

**Comment by geoffbullen** [comment]
Keen to know your thoughts/plans regarding android as well.

**Comment by EddyVerbruggen** [comment]
Google released a 1.0 last week so I'm now trying to integrate that. Progress is slow because I have many other things on my plate. But I am working on it.

**Comment by robinbonnes** [comment]
Creating apps with ARCore makes more sense now, as they supported more devices in the ARCore Preview 2: https://developers.google.com/ar/discover/#supported_devices

For the developers that want to support even more devices, they can try patch the SDK. However I think it won't be accepted in the Play Store:

- https://github.com/thestr4ng3r/arcore-patch
- https://github.com/tomthecarrot/arcore-for-all

**Comment by lee-devopa** [comment]
What's the progress on this? Looks really interesting just a shame the android support is lacking

**Comment by Emanuele-Fortunati** [comment]
@EddyVerbruggen hey Eddy hows going with Android support? The latest ARCore supports many more features and devices, so it would make sense to bring more stuff into the plugin too. I'm happy to contribute, I'd need the original java files though. Please let me know.

**Comment by EddyVerbruggen** [comment]
@Emanuele-Fortunati I you're still interested to contribute: please see the latest state of this repo and especially issue #37. I'm pretty close to releasing version 1.0.0 which has much much better support for Android.

**Comment by Emanuele-Fortunati** [comment]
@EddyVerbruggen That's really great news: I've had a look at #37 and I'm happily impressed. 
In the meanwhile, I've added to your plugin the possibility of playing Lottie's Animations, Views and multi-layers videos / animations. The code is a bit of a mess up as I had to release it on the rush, but once 1.0 is out, I'll update it with my code (a cleaner version) and if you wish I can do a PR ;)
Thanks for the great job!
